packages/server/assets/workspacesCore/typedefs/workspaces.graphql

```diff
@@ -88,3 +157,9 @@ extend type AdminQueries {
 input UserWorkspacesFilter {
   search: String
 }
+
+enum WorkspaceRole {
```

Comment:
would be cool to create something similar to `@hasStreamRole` for workspace role checks

Reply:
+1 agreed, created a linear issue for this